README.md

```diff
@@ -16,9 +16,9 @@ The Test Harness incorporates Test Runners that do the actual running of the aut
 - output_curie: curie checked for in the results
 
 ## How to use:
-The Test Harness is a CLI that you need to install:
+The Test Harness is a CLI that you need to install (preferably within a suitable Python Virtual Environment):
 - `pip install -r requirements.txt` to install normal dependencies
-- `pip install -r requirements-runners.txt` to install the Test Runners
+- `pip install -r requirements-runners.txt` (or **requirements-runners-win64.txt** using Python 3.12 for _jq_ module compatibility) to install the Test Runners 
```

Comment:
I'm not actually a huge Windoze fan myself, just that a Dell workstation running it felt like a good choice the time I needed a new home workstation (I did look at Apple Mac as an alternative but...much higher cost/benefit ratio).

Again, the workarounds here are a bandage .. For sure, we can add a separate README patch recipe section about Windoze.

I spend a couple of hours one day trying to find a Windoze solution to the *nix jq dependency. Found a partial answer which is still problematic.

I think you mentioned that perhaps, the **jq** wasn't a strict necessity wherever it is used; otherwise, we will need to somehow build acceptable Windoze jq wheels (cached in the project) as a workaround.

Reply:
I didn't want to bring it up because it seemed you had put some significant work into getting jq to work, but yes, it is currently not being used (it is used in the UI Test Runner that I made, but we are currently not/can't test the UI at this time).

Reply:
HI @maximusunc, I simply attempted to troubleshoot the issue, only partly successfully. If we remove jq from the dependencies (for now), the problem goes away, right?